lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -150,45 +141,60 @@ def read_from_ssm(key: str) -> str:
-    auth_token = read_from_ssm(os.getenv('vetext_api_auth_ssm_path'))
-    logger.info("Retrieved AuthToken from SSM")
-    
-    headers = {
-        'Content-type': 'application/json',
-        'Authorization': 'Basic ' + auth_token
-    }
-
-    body = {
-            "accountSid": request_body.get("AccountSid", ""),
-            "messageSid": request_body.get("MessageSid", ""),
-            "messagingServiceSid": request_body.get("MessagingServiceSid", ""),
-            "to": request_body.get("To", ""),
-            "from": request_body.get("From", ""),
-            "messageStatus": request_body.get("SmsStatus", ""),
-            "body": request_body.get("Body", "")
+    try:
+        connection = http.client.HTTPSConnection(os.getenv('vetext_api_endpoint_domain'),  context = ssl._create_unverified_context())
+        logger.info("generated connection to VeText")
+
+        # Authorization is basic token authentication that is stored in environment.
+        auth_token = read_from_ssm(os.getenv('vetext_api_auth_ssm_path'))
+        logger.info("Retrieved AuthToken from SSM")
+        
+        headers = {
+            'Content-type': 'application/json',
+            'Authorization': 'Basic ' + auth_token
         }
 
-    json_data = json.dumps(body)
+        body = {
+                "accountSid": request_body.get("AccountSid", ""),
+                "messageSid": request_body.get("MessageSid", ""),
+                "messagingServiceSid": request_body.get("MessagingServiceSid", ""),
+                "to": request_body.get("To", ""),
+                "from": request_body.get("From", ""),
+                "messageStatus": request_body.get("SmsStatus", ""),
+                "body": request_body.get("Body", "")
+            }
 
-    logger.info("Making POST Request to VeText using: " + os.getenv('vetext_api_endpoint_domain') + os.getenv('vetext_api_endpoint_path'))
-    logger.debug(f"json dumps: {json_data}")
-    
-    connection.request(
-        'POST',
-        os.getenv('vetext_api_endpoint_path'),
-        json_data,
-        headers)
+        json_data = json.dumps(body)
 
-    response = connection.getresponse()
-    
-    logger.info(f"VeText call complete with response: {response.status}")
-    logger.debug(f"VeText response: {response}")
+        logger.info("Making POST Request to VeText using: " + os.getenv('vetext_api_endpoint_domain') + os.getenv('vetext_api_endpoint_path'))
+        logger.debug(f"json dumps: {json_data}")
+        
+        connection.request(
+            'POST',
+            os.getenv('vetext_api_endpoint_path'),
+            json_data,
+            headers)
+
+        response = connection.getresponse()
+        
+        logger.info(f"VeText call complete with response: {response.status}")
+        logger.debug(f"VeText response: {response}")
+
+        if response.status == 200:
+            return response        
+
+        logger.info("VeText call failed.")
```

Comment:
This is redundant to the logging in the except blocks, and I think it should be "error" rather than "info".